lib/matplotlib/axes/_axes.py

```diff
@@ -4480,7 +4480,6 @@ def scatter(self, x, y, s=None, c=None, marker=None, cmap=None, norm=None,
         path = marker_obj.get_path().transformed(
             marker_obj.get_transform())
         if not marker_obj.is_filled():
-            edgecolors = 'face'
```

Comment:
I think this change requires a modification of the docstring entry for edgecolors, which states, "For non-filled markers, the *edgecolors* kwarg is ignored and forced to 'face' internally."

Reply:
Done.